app/src/main/java/com/kickstarter/libs/Koala.java

```diff
@@ -493,4 +493,11 @@ public void trackOpenedExternalLink(final @NonNull Project project, final @NonNu
 
     this.client.track(KoalaEvent.OPENED_EXTERNAL_LINK, props);
   }
+
+  // DEEP LINK
+  public void trackUserActivity() {
```

Comment:
we tend to name these Koala event functions exactly the event they're tracking; since `UserActivity` is kind of vague, maybe we can be Xtra and name this `trackContinueUserActivityAndOpenedDeepLink` or something of the sorts!